### PR TITLE
Add volume up/down buttons, glyphs cleanup, buttons cleanup

### PR DESCRIPTION
**Pull request by Havner**

### CODE REVIEW

**Comment by Havner** [comment]
This probably shouldn't be exactly pushed in the current form. Awaiting comments.

1. Should the vol up/down buttons be added to IDevice or per device basis?
2. Should remapper have a separate section for those buttons instead of OEM like they are now?

ad. 2.
vol1

File: HandheldCompanion/Controllers/NeptuneController.cs [hunk 1]
```diff
@@ -87,21 +87,16 @@ public NeptuneController(PnPDetails details)
...
             SupportedButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.RightPadClick, ButtonFlags.RightPadTouch, ButtonFlags.RightPadClickUp, ButtonFlags.RightPadClickDown, ButtonFlags.RightPadClickLeft, ButtonFlags.RightPadClickRight });
 
             // Specific buttons that shouldn't be mappable
-            VirtualButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.L4, ButtonFlags.R4, ButtonFlags.L5, ButtonFlags.R5 });
-            VirtualButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.LeftPadClick, ButtonFlags.LeftPadTouch, ButtonFlags.LeftPadClickUp, ButtonFlags.LeftPadClickDown, ButtonFlags.LeftPadClickLeft, ButtonFlags.LeftPadClickRight });
-            VirtualButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.RightPadClick, ButtonFlags.RightPadTouch, ButtonFlags.RightPadClickUp, ButtonFlags.RightPadClickDown, ButtonFlags.RightPadClickLeft, ButtonFlags.RightPadClickRight });
+            VirtualButtons = new(SupportedButtons);
```
Comment (Valkirie) [inline HandheldCompanion/Controllers/NeptuneController.cs:99]: Why would you add all buttons to the virtual buttons list ?

Comment (Havner) [reply HandheldCompanion/Controllers/NeptuneController.cs:99]: I'm not entirely sure what this list is for. I've checked references, but it seems the idea is that those keys are not target for the remapper.

In any case all the buttons were already on this list. I just added OEM and Vol keys there. So in general I did not change anything, just simplified the code.

What buttons should be on the VirtualButtons list by design?

Comment (Havner) [reply HandheldCompanion/Controllers/NeptuneController.cs:99]: Ok, I know what you mean now.

File: HandheldCompanion/Controllers/NeptuneController.cs [hunk 2]
```diff
@@ -86,22 +86,20 @@ public NeptuneController(PnPDetails details)
             DrawControls();
             RefreshControls();
 
+            List<ButtonFlags> ExtraSteamDeckButtons = new();
+            ExtraSteamDeckButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.OEM1, ButtonFlags.VolumeUp, ButtonFlags.VolumeDown });
```
Comment (Valkirie) [inline HandheldCompanion/Controllers/NeptuneController.cs:90]: I'd move VolumeUp and VolumeDown to IController VirtualButtons list. (OEM1 is already there).

**Pull request merged**